README.md

```diff
@@ -1,55 +1,29 @@
-# Random Walk Simulation
+# Instructions for the random walker scipt:
```

Comment:
Sadly your test images are not visible in your repository, might be a problem on my side, but would still be nice if you checked this :)

Reply:
the links to the screenshots were refering to a side-branch, not the main branch. Now that the side-branch is gone they got destroyed. Thanks!